packages/next-app/src/app/pool/[poolId]/loading.tsx

```diff
@@ -0,0 +1,70 @@
+export default function Loading() {
+  return (
+    <div className="h-full flex-1 py-5 text-white">
+      <div className="w-full bg-gray-900">
```

Comment:
I noticed we have tailwind colors and radix colors 🤔 which do you think we should commit to? No need to go back and refactor everything, but wanted to get your perspective

Reply:
I don't know well the radix colors but seems to be a good option. We can ask about the Nichollas option as well